src/components/shared/Slider/index.tsx

```diff
@@ -13,21 +16,46 @@ interface Props {
   step?: number;
 }
 
-const Slider: FC<Props> = () => {
+const Slider: FC<Props> = ({ maxValue = 100, minValue = 0 }) => {
+  const sliderRef = useRef<any>();
+  const [sliderWidth, setSliderWidth] = useState(0);
+  const [layoutX, setLayoutX] = useState(0);
+  const [positionX, setPositionX] = useState(0);
+
+  const getPercentagePosition = (): number => {
+    const percentage = (positionX / sliderWidth) * 100;
+    if (percentage <= 0) return 0;
+    else if (percentage >= 100) return 100;
+    else return percentage;
+  };
+
   const panResponder = React.useMemo(
     () =>
       PanResponder.create({
-        onStartShouldSetPanResponder: () => true,
-        onPanResponderGrant: (evt, gestureState) => {
-          console.log(gestureState);
+        onMoveShouldSetPanResponder: () => true,
+        onPanResponderMove: (evt, gestureState) => {
```

Comment:
I don't know about `PanResponder`. But I guess that `PanResponder` is [Track]'s role.
Any reason that you added `onPanResponderMove` here?

Reply:
I think Track and Thumb both of them should know the position user touch. So I tried to calculate the position in Slide and pass it to Track and Thumb.

Reply:
We should talk a little bit about the roles of [Track], [Thumb] and [Slider] here, to share and sync what we develop.
What do you guys think about the components?

I think they are like:

| Slider | Rail |Track | Thumb | Label
|-------|-----| -------|--------|-----|
| combines all of the features. | draws marks and passes up the positions of marks. | moves the position of [Thumb] by user gesture. | an animatable icon by touching. | a text with location icon |

I guess that @YongPilMoon thinks they are like:

| Slider |Track | Thumb |
|-------|-------|--------|
| moves the position of [Thumb] by user gesture. And it passes the position value to [Track] and [Thumb]. | draws the track style by the passed position value. | locates an animatable icon to the passed position. |

How to imagine the components? :)
@YongPilMoon @daheeahn @tony3918 @hankkuu